docstring_parser/parser/google.py

```diff
@@ -0,0 +1,135 @@
+"""Google-style docstring parsing."""
+
+import inspect
+import re
+
+from .common import ParseError, Docstring, DocstringMeta
+
+_sections = {
+    'Arguments': 'param',
+    'Args': 'param',
+    'Parameters': 'param',
+    'Params': 'param',
+    'Raises': 'raises',
+    'Exceptions': 'raises',
+    'Except': 'raises',
+    'Attributes': None,
+    'Example': None,
+    'Examples': None,
+    'Returns': 'returns',
+    'Yields': 'returns',
+}
+_titles = '|'.join('(%s)' % t for t in _sections)
```

Comment:
Please use `re.compile` here and full regex.
Please name this `_titles_re`.
Consider using `re.escape` for the section names (might be an overkill).

Reply:
Is 
```python
_titles_re = re.compile(
    '^(' + '|'.join('(%s)' % t for t in _sections) + '):',
    flags=re.M)
```
what you were expecting? Also `re.escape` seems to be particular for escaping bad characters in an option group, of which the section names don't have any

Reply:
Yes. But I don't see how `re.M` is needed?

Reply:
I use the multi-line flag to check each line if it starts with the section title. I could replace `'^'` with `'\n'` in the expression and achieve the same result

Reply:
Okay. Both are fine :)

Reply:
Turns out the new-line method actually causes issues, so using the multi-line flag is the way to go